### ISSUE TITLE
<1040px side nav overlay makes page unusable

### ISSUE THREAD

**Issue by dakahn** [carbon-design-system/carbon-website#2285]
## Detailed description

> Describe in detail the issue you're having.

At <1040px the side nav overlay is shown and the user cannot interact with any page elements.

> What browser are you working in?

Firefox latest and Chrome latest on Windows 10

![image](https://user-images.githubusercontent.com/40970507/117197811-e18d1880-adad-11eb-86e6-d925f8e86883.png)

**Comment by tw15egan** [comment]
@dakahn can you try doing a hard refresh? I put in a fix for this in https://github.com/carbon-design-system/carbon-website/pull/2187 and I'm not seeing it on my end (I'm on Mac though)

**Comment by dakahn** [comment]
Yep, you're right. Was probably happening on an out of date local
